react/package.json

```diff
@@ -5,10 +5,10 @@
     "react-color": "^2.17.0",
     "react-datepicker": "^2.0.0",
     "react-dropzone": "^10.1.8",
+    "react-focus-lock": "^2.3.1",
```

Comment:
This lib is a [bit big for what it does](https://bundlephobia.com/result?p=react-focus-lock@2.3.1), maybe you can implement it. [Here is an example](https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/Modal/TrapFocus.js)